utils/python/CIME/SystemTests/pem.py

```diff
@@ -12,119 +12,29 @@
 from CIME.XML.standard_module_setup import *
 from CIME.case_setup import case_setup
 import CIME.utils
-from CIME.SystemTests.system_tests_common import SystemTestsCommon
+from CIME.SystemTests.system_tests_compare_two import SystemTestsCompareTwo
 
 logger = logging.getLogger(__name__)
 
-class PEM(SystemTestsCommon):
+class PEM(SystemTestsCompareTwo):
 
     def __init__(self, case):
         """
         initialize a test object
         """
-        SystemTestsCommon.__init__(self, case)
-
-    def build_phase(self, sharedlib_only=False, model_only=False):
-        """
-        Build two cases.  Case one uses defaults, case2 uses half the number of threads
-        and tasks. This test will fail for components (e.g. pop) that do not reproduce exactly
-        with different numbers of mpi tasks.
-        """
-        exeroot = self._case.get_value("EXEROOT")
-        cime_model = CIME.utils.get_model()
-
-        machpes1 = os.path.join("LockedFiles","env_mach_pes.PEM1.xml")
-        if ( os.path.isfile(machpes1) ):
-            shutil.copy(machpes1,"env_mach_pes.xml")
-        else:
-            logging.warn("Copying env_mach_pes.xml to %s"%(machpes1))
-            shutil.copy("env_mach_pes.xml", machpes1)
+        SystemTestsCompareTwo.__init__(self, case, True)
 
+    def _common_setup(self):
         self._case.set_value("SMP_BUILD","0")
```

Comment:
I strongly suspect that the setting of SMP_BUILD to 0 is no longer needed if you use this clone-based infrastructure. Though I can't be 100% sure since I don't actually understand what this setting does :-)